README.md

```diff
@@ -69,6 +69,12 @@ Setting up Cockroach is easy, but starting a test node is even easier. All it ta
 ./cockroach start --dev &
 ```
 
+Verify that you're up and running by visiting the cluster UI. If you're running
+without Docker (or on Linux), you'll find it at
+[localhost:26257](http://localhost:26257); for OSX, things are a little more
```

Comment:
this wording suggests that you always need to go to `docker-machine ip default` on OSX (regardless of docker). of course that would be nonsense, but y'know.